grails-app/services/org/bbop/apollo/BigwigService.groovy

```diff
@@ -79,19 +79,14 @@ class BigwigService {
         for (Integer i = actualStart; i < actualStop; i += stepSize) {
             JSONObject globalFeature = new JSONObject()
             Integer endStep = i + stepSize
-            globalFeature.put("start", i+projectionSequence.projectedOffset)
-            globalFeature.put("end", endStep+projectionSequence.projectedOffset)
+            globalFeature.put("start", i + projectionSequence.projectedOffset)
+            globalFeature.put("end", endStep + projectionSequence.projectedOffset)
             Integer originalStart = projection.unProjectValue(i)
             Integer originalEnd = projection.unProjectValue(endStep)
             edu.unc.genomics.Contig innerContig = bigWigFileReader.query(projectionSequence.name, originalStart, originalEnd)
             Integer value = innerContig.mean()
 
-            if (value >= 0) {
-//                        // TODO: this should be th mean value
-                globalFeature.put("score", value)
-            } else {
-                globalFeature.put("score", 0)
-            }
+            globalFeature.put("score", value >= 1 ?: 1)
```

Comment:
Doing this ends up with red segments with NaN instead of 0.

Instead, it should be:
```
            if (value >= 1) {
                globalFeature.put("score", value)
                featuresArray.add(globalFeature)
            }
```

We shouldn't be adding the globalFeature to the featuresArray if its not >= 1.